styleguide/foundations/icon-font.stories.mdx

```diff
@@ -72,3 +87,36 @@ Do not forget to prefix the given class name with `cads-icon_`.
     <span>undo</span>
   </li>
 </ul>
+
+## Alignment
+
+When an icon is right next to the text, they should be centre-aligned and have 4px spacing in-between.
+
+<img src={iconExample} style={{width: '350px'}} />
+
+## Contribute
+
+### Considerations before creating a new icon
+
+Before trying to create a new icon, consider if:
+- The new icon serves a need(s). If in doubt, you probably don’t need it
+- The new icon would be used multiple times throughout the products
+- The new icon has a clear meaning
+
+Propose the new icons to the design community and discuss details and requirements. This will help us to have a transparent process for creating new assets, maintaining a minimal and high quality icon set in our Design system. 
+
+### Making an icon
```

Comment:
Interesting. Some really great principles in here but/and I think some of these steps might be unnecessary in the figma stage as we could handle it in code. I have a couple of clarifying questions first though, will ask on slack.

Reply:
fWe chatted through this the notes all make sense now. Suggested adding a visual example of the pixel grid requirements to show what to look out for, as well as a good and bad example of SVG output.

Reply:
I've updated this section based on the suggestions above. I think the images for the pixel section work well; however, I'm not too sure about the SVG ones. They feel a bit overkill to me, what do you think?